script.js

```diff
@@ -9,3 +9,19 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+
+
+var colors = ["red", "yellow", "green"];
+console.log(colors[2]);
+
+for(i = 0; i <colors.length; i++) {
+  console.log("My favorite color is " + colors[i]);
+}
+
+let instructor = {
+  name: "Adrian",
+  cohortsTaught: 7,
+  hobby: "knitting"
+};
+
+instructor.facialHair = "beard and mustache";
```

Comment:
The trick here is that the key had to be spelled `facial-hair` (i.e., with a hyphen). When you introduce syntax like that into a key name, you need to use bracket notation, like so...

```js
instructor["facial-hair"] = true
```